api/can_api_v2_definition.py

```diff
@@ -261,6 +312,7 @@ class RegionSummary(base_model.APIBaseModel):
     metrics: Metrics = pydantic.Field(...)
     riskLevels: RiskLevels = pydantic.Field(..., description="Risk levels for region.")
     actuals: Actuals = pydantic.Field(...)
+    annotations: Annotations = pydantic.Field(...)
```

Comment:
What are your thoughts on making annotations here a map of `[FieldName: MetricAnnotations]`? Actually I'm going to read the rest of the PR and see how the rest works first ;)

Reply:
Ah cool, yeah that's correct - I think it'll be cleanest to not include fields that have have no annotations.  so the actual result would only have the keys of annotations that are there

Reply:
Most fields have a source so most will have a `MetricAnnotations`.